app/src/main/res/layout/item_card_view.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView android:id="@+id/cvContact"
+                                    xmlns:android="http://schemas.android.com/apk/res/android"
+                                    xmlns:app="http://schemas.android.com/apk/res-auto"
+                                    android:layout_width="match_parent"
+                                    android:layout_height="@dimen/default_card_view_height"
+                                    app:cardCornerRadius="15dp"
+                                    app:cardElevation="3dp"
+                                    app:cardMaxElevation="5dp"
+                                    app:cardUseCompatPadding="true">
+
+    <RelativeLayout
```

Comment:
Please, use Constraint layout. You do not need other ViewGroup.

Reply:
Does that mean that I'm expected to use it under any circumstances, even if the layout is small and contains only few views?